src/extensions/surveys/surveys-utils.tsx

```diff
@@ -559,20 +559,45 @@ export const createShadow = (styleSheet: string, surveyId: string, element?: Ele
     return shadow
 }
 
-export const sendSurveyEvent = (
-    responses: Record<string, string | number | string[] | null> = {},
-    survey: Survey,
+type SendSurveyEventArgs = {
+    responses: Record<string, string | number | string[] | null>
+    survey: Survey
     posthog?: PostHog
-) => {
+    surveyCompleted?: boolean
+    surveyResponseInsertID?: string
+}
+
+export const sendSurveyEvent = ({
+    responses,
+    survey,
+    posthog,
+    surveyCompleted,
+    surveyResponseInsertID,
+}: SendSurveyEventArgs) => {
     if (!posthog) {
         logger.error('[survey sent] event not captured, PostHog instance not found.')
         return
     }
     localStorage.setItem(getSurveySeenKey(survey), 'true')
 
+    // Don't send partial response if survey is not completed and disable_survey_partial_response is true
```

Comment:
will `disable_survey_partial_response` be default'ed to `false`? or should that be an opt in feature?

Reply:
no lonver relevant, setting is now per survey